st2actions/st2actions/policies/concurrency.py

```diff
@@ -62,7 +62,7 @@ def _apply_before(self, target):
             status = self._get_status_for_policy_action(action=self.policy_action)
 
         # Update the status in the database but do not publish.
```

Comment:
`# Update the status in the database and publish.`

Applies to next file as well.